### ISSUE TITLE
Consolidating tags

### ISSUE THREAD

**Issue by khoroshevskyi** [pepkit/pephub#310]
Right now, we treat `namespace/name:tag` as completely different projects.

What if we consolidate all `name:tag` entries to `name`, and from the project level, we retrieve the desired tag of the project?

e.g.
![image](https://github.com/pepkit/pephub/assets/41573628/d8231e8a-ba2c-41ea-9060-19db7414c163)

This 2 project in namespace page would be just ONE project.

Also, if user doesn't know tag of the project, then user can access project, even if there is only one existing tag.

**Comment by nleroy917** [comment]
Yeah this makes the most sense to me. This is actually how dockerhub does it. Check out the [pephub docker image](https://hub.docker.com/search?q=pephub)

<img width="987" alt="image" src="https://github.com/pepkit/pephub/assets/41063083/3fa67b26-2562-4339-8cb8-00cdafc58e0a">

It has a `tags` tab:

<img width="1259" alt="image" src="https://github.com/pepkit/pephub/assets/41063083/f6dfedd6-611a-4681-a23f-6259df9b4da0">

So maybe we add a dropdown on a project page and just let the user pick a tag

**Comment by nleroy917** [comment]
An idea is a dropdown where a user can select the tag (defaults to `default`) unless query param is specified:

<img width="1728" alt="image" src="https://github.com/pepkit/pephub/assets/41063083/1537a0bc-095e-442e-9080-60c317461584">

**Comment by nleroy917** [comment]
While I'm looking at it, I propose getting rid of the blue schema tag it doesn't seem to add anything and bloats it a bit

**Comment by khoroshevskyi** [comment]
and instead of tag, annotation return model will have tags, with available tags

```python
class AnnotationModel(BaseModel):
    """
    Project Annotation model. All meta metadata
    """

    namespace: Optional[str]
    name: Optional[str]
    **tags: Optional[str]**
    is_private: Optional[bool]
    number_of_samples: Optional[int]
    description: Optional[str]
    last_update_date: Optional[str]
    submission_date: Optional[str]
    digest: Optional[str]
    pep_schema: Optional[str]
    pop: Optional[bool] = False
    stars_number: Optional[int] = 0
    forked_from: Optional[Union[str, None]] = None
```

**Comment by nleroy917** [comment]
yeah I like this idea -- this makes a lot of sense. Can we leverage this to get rid of these query params too?

<img width="254" alt="image" src="https://github.com/pepkit/pephub/assets/41063083/34c38b89-3fe3-429f-9f86-f59818c7d29f">

Here is an example route:
```python
@project.get(
    "/views",
    summary="get list of views for a project",
    tags=["views"],
    response_model=ProjectViews,
)
def get_views(
    namespace: str,
    project: str,
    tag: str = DEFAULT_TAG,
    agent: PEPDatabaseAgent = Depends(get_db),
):
    return agent.view.get_views_annotation(namespace, project, tag=tag)
```

**Comment by nleroy917** [comment]
@khoroshevskyi What do you think about this one? Any progress made to the backend?
